src/Symfony/Bundle/FrameworkBundle/Command/CacheClearCommand.php

```diff
@@ -33,10 +30,6 @@ protected function configure()
     {
         $this
             ->setName('cache:clear')
-            ->setDefinition(array(
-                new InputOption('no-warmup', '', InputOption::VALUE_NONE, 'Do not warm up the cache'),
```

Comment:
A similar option in a form was used as a bc layer between 2.8->3.0, it was kept in 3.0 and deprecated in 3.1 only.
So I would just ignore it and deprecate it in 4.1.